projects/driver_controls/src/turn_signal_fsm.c

```diff
@@ -31,22 +32,19 @@ FSM_STATE_TRANSITION(state_right_signal) {
 // Turn signal FSM output function
 
 static void prv_state_output(FSM *fsm, const Event *e, void *context) {
-  InputEventData data = { 0 };
-
-  // woah this was very wrong
-  data.components.data = e->data;
+  TurnSignalFSMState turn_signal_state = 0;
```

Comment:
This should be initialized to an enum, not an integer.

Reply:
Done